chapter9.adoc

```diff
@@ -0,0 +1,346 @@
+[[chapter9]]
+[[Smsdqos]]
+== Capacity and Bandwidth QoS Register Interface for Supervisor Domains
+
+Quality of Service (QoS) is defined as the minimal end-to-end performance
+that is guaranteed in advance by a service level agreement (SLA) to a
+workload. A workload may be a single application, a group of applications,
+a virtual machine, a group of virtual machines, or a combination of those.
+
+The RISC-V Capacity and Bandwidth QoS Register Interface (CBQRI) cite:[CBQRI]
+specification specifies mechanisms to a) associate QoS identifiers with
+requests made by workloads to resources controlled by capacity and bandwidth
+controllers, b) memory-mapped register interface for capacity allocation and
+capacity usage monitoring in capacity controllers in shared resources such as
+shared caches, c) memory-mapped register interface for bandwidth allocation
+and bandwidth usage monitoring in bandwidth controllers in shared resources
+such as memory controllers.
+
+The Ssqosid extension cite:[SSQOSID] provides a read/write S/HS-mode register
+(`srmcfg`) to configure QoS Identifiers to be used with requests made by the
+hart to shared resources. The CBQRI specification specifies the IOMMU extension
+to associate QoS Identifiers with requests made by devices to shared resources.
+
+With supervisor domains, the resource accesses from a SD or the RDSM must not
+be observable using the resource usage monitors to entities that are not in
+their TCB.  The resource allocations for a SD or the RDSM must not be affected
+by entities that are not in their TCB.
```

Comment:
move security requirements up to the front to explain the rationale for the sub-extension

Reply:
I think this is in a  logical place. The paragraph 1 and 2 and 3 provide an introduction to what CBQRI is. This parapgraph - paragraph 4 - explains why this extension is needed. The para 1, 2, and 3 are needed to appreciate para 4.

Reply:
It came up as a question - at least a heading such as “security objectives “ would be nice to add to avoid readers wondering

Reply:
Please see updated.